pkg/gatherers/clusterconfig/node_logs_test.go

```diff
@@ -146,14 +147,30 @@ func Test_requestNodeLog(t *testing.T) {
 				tail:   10,
 				unit:   "test",
 			},
-			want: r.RequestURI("/path/to/something").Param("tail", "10").Param("unit", "test").Verb("GET"),
+			want: c.Get().
+				RequestURI("/path/to/something").
+				Param("tail", "10").
+				Param("unit", "test").
+				SetHeader("Accept", "text/plain, */*").
+				SetHeader("Accept-Encoding", "gzip"),
 		},
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			if got := requestNodeLog(tt.args.client, tt.args.uri, tt.args.tail, tt.args.unit); !reflect.DeepEqual(got, tt.want) {
-				t.Errorf("requestNodeLog() = %v, want %v", got, tt.want)
-			}
+			got := requestNodeLog(tt.args.client, tt.args.uri, tt.args.tail, tt.args.unit)
```

Comment:
>reflect.DeepEqual(got, tt.want)

This is failing with the update to Kubernetes client-go 1.24. Not 100% sure, but I guess that it fails to compare the [`retryFn`](https://github.com/kubernetes/client-go/blob/v0.24.0/rest/request.go#L122) which was previously defined as the [`WithRetry interface`](https://github.com/kubernetes/client-go/blob/v0.23.1/rest/request.go#L114).

Another option is to remove this test (and probably the `requestNodeLog` function as well) since the test doesn't have much value IMO.